### PR TITLE
fit omega models

### PR DESCRIPTION
**Pull request by zen-juen**
Now model_parameters() can run on omega models!
```
omega <- psych::omega(mtcars, nfactors = 3)
params <- model_parameters(omega)
params

#> Composite | Omega (total) | Omega (hierarchical) | Omega (group)
#>----------------------------------------------------------------
#> g         |          0.97 |                 0.57 |          0.26
#>F1*       |          0.90 |                 0.31 |          0.59
#>F2*       |          0.91 |                 0.69 |          0.22
#>F3*       |          0.87 |                 0.60 |          0.28
```
Also accompanied with a summary method:
```
summary(params)
#>Composite | Total Variance (%) | Variance due to General Factor (%) | Variance due to Group Factor (%)
#>------------------------------------------------------------------------------------------------------
#> g         |              97.28 |                              56.64 |                            26.42
#>F1*       |              90.12 |                              31.07 |                            59.05
#>F2*       |              91.37 |                              69.32 |                            22.04
#>F3*       |              87.36 |                              59.65 |                            27.71
```

Will be nice to also have a plot() to produce a cor.plot heat map of the omega indexes!

### CODE REVIEW

**Comment by strengejacke** [comment]
Thanks, looks great! I wonder if there are shorter column names, though? And please add yourself as contributor to the description file.

File: R/model_parameters.psych.R [hunk 1]
```diff
@@ -140,16 +149,43 @@ model_parameters.fa <- model_parameters.principal
...
+  table_om <- model$omega.group
+  colnames(table_om) <- c("Omega_Total", "Omega_Hierarchical", "Omega_Group")
+  table_om$Composite <- row.names(table_om)
+  row.names(table_om) <- NULL
+  table_om <- table_om[c("Composite", names(table_om)[names(table_om) != "Composite"])]
+  
+  # Get summary: Table of Variance
+  table_var <- as.data.frame(unclass(model$omega.group))
+  table_var$Composite <- rownames(model$omega.group)
+  table_var$Total <- table_var$total*100
```
Comment (strengejacke) [inline R/model_parameters.psych.R]: Please use whitespaces around `*` etc.: `table_var$total*100` -> `table_var$total * 100`

Comment (DominiqueMakowski) [reply R/model_parameters.psych.R]: @zen-juen you can run `styler::style_file("filepath")` to automatically nicely format a given file

Comment (strengejacke) [reply R/model_parameters.psych.R]: I don't like these tools, as they do too much in my opinion. Would be sufficient to activate style diagnostic (https://support.rstudio.com/hc/en-us/articles/205753617-Code-Diagnostics) and then just hit Ctrl+Shift+A on selected lines. :-)

Comment (DominiqueMakowski) [reply R/model_parameters.psych.R]: I think they are ways of parametrizing styler, but yeah I can understand (although as for now I've been fairly satisfied with styler's defaults ☺️)

**Comment by strengejacke** [comment]
Does this PR do any harm? I don't think so. Need to submit _parameters_ within the next hours, so we must decide whether to merge this or not.

**Comment by DominiqueMakowski** [comment]
can merge 👍

**Pull request merged**